src/ngx-smart-modal/src/components/ngx-smart-modal.component.ts

```diff
@@ -72,7 +73,9 @@ export class NgxSmartModalComponent implements OnInit, OnDestroy {
   constructor(
     private _renderer: Renderer2,
     private _changeDetectorRef: ChangeDetectorRef,
-    private _ngxSmartModalService: NgxSmartModalService
+    private _ngxSmartModalService: NgxSmartModalService,
+    @Inject(DOCUMENT) private _document: Document,
+    @Inject(PLATFORM_ID) private _platformId: Object
```

Comment:
TravisCI signals this `Object` must be `object`.

Reply:
fixed